src/accumulate.c

```diff
@@ -31,7 +31,11 @@ void accumulate_sse(const float *in, uint8_t *out, uint32_t n) {
     y = _mm_mul_ps(y, _mm_set1_ps(255.0f));
     __m128i z = _mm_cvtps_epi32(y);
     z = _mm_shuffle_epi8(z, mask);
+#ifdef _MSVC
+    _mm_store_ss((float *)&out[i], _mm_castsi128_ps(z));
+#else
     _mm_store_ss((float *)&out[i], (__m128)z);
+#endif
```

Comment:
This doesn't need to be conditional, the casting function works fine on Mac OS.

Thanks for the fix, it's appreciated!